### PR TITLE
chore: Create Github Action that requires PR comments to be resolved.

### PR DESCRIPTION
**Pull request by melink14**

### CODE REVIEW

**Review by melink14 — COMMENTED** [review]
please fix.

File: .github/workflows/resolve-comments.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: No unresolved comments
+on:
+  pull_request_review_comment:
+
+jobs:
```
Comment (Stephie) [inline .github/workflows/resolve-comments.yml:5]: boop

File: .github/workflows/resolve-comments.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  readResolutionStatus:
```
Comment (Stephie) [inline .github/workflows/resolve-comments.yml:6]: neat!

**Review by Stephie — APPROVED** [review]

**Review by melink14 — COMMENTED** [review]
not a comment?

**Pull request merged**